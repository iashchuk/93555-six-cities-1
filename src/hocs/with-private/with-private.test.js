import React from "react";
import { Redirect } from "react-router-dom";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import withPrivate from "./with-private.js";

configure({ adapter: new Adapter() });

const isAuthenticated = false;
const match = {
  url: `/favorites`
};

const Component = () => <Redirect to="/login" />;
const WithPrivate = withPrivate(Component);

describe(`withPrivate component`, () => {
  it(`renders correctly`, () => {
    const component = shallow(
        <WithPrivate match={match} isAuthenticated={isAuthenticated} />
    );
    expect(component).toMatchSnapshot();
  });
});
