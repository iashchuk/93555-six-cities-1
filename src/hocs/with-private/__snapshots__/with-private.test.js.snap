// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withPrivate component renders correctly 1`] = `ShallowWrapper {}`;
